README.md

```diff
@@ -0,0 +1,45 @@
+# Galoy CLI
+Galoy CLI is a Rust-based CLI client that can interact with the Galoy backend using GraphQL queries and mutations.
+
+## Installation
+To install Galoy CLI locally and set up a local environment:
+
+1. Clone the repository using git clone `https://github.com/GaloyMoney/galoy-cli.git`.
+2. Install Rust in your local machine and run cargo build to build all binary and library targets of the selected packages.
+3. Run cargo run command to run all tests of the Galoy CLI repository and see the usage, commands, and options available to interact.
+4. Interact with the CLI yourself to become familiar with it. After building, try the following command:
+
+`GALOY_API=https://api.staging.galoy.io/graphql ./target/debug/galoy-client getinfo`
+This command will retrieve the global values from the Galoy instance.
+
+You can also test if the "GALOY CLI CAPTCHA SERVER" is running or not by running the following command:
+
+`GALOY_API=https://api.staging.galoy.io/graphql ./target/debug/galoy-client request-phone-code +16505554321`
+
+## Usage
+To use the Galoy CLI, you need to run it with the desired command and options. 
+
+### Commands:
+getinfo:&#9; &#9; &#9;Get global values from the instance. <br/>
+default-wallet:     Get WalletId for an account. <br/>
+me:                 Execute Me query. <br/>
+send-intraledger:   Do an intraledger transaction. <br/>
+request-phone-code: Request a code from a Phone number. <br/>
+login:              Get JWT of an account. <br/>
+batch:              Execute a batch payment.
+
+To see the available options for each command, run galoy-client <COMMAND> --help.
+
+
+### Options
+The available options for the Galoy CLI are:
+
+-a, --api <API>:   Set the API URL (default: http://localhost:4002/graphql) <br/>
+-d, --debug:       Enable debug mode<br/>
+-j, --jwt <JWT>:   Set the JWT for authorization<br/>
```

Comment:
I think jwt is not a valid term for the access token we get when we login ? I even got error when I tired to set in access token as JWT in the cli

Reply:
I am also getting error. basically showing "jwt syntax issue".

Reply:
yup long story short, there should no longer be a JWT stored, but a kratos token. it should also be passed as a `Authorization` header to the backend.

Reply:
it probably means this command line should be update. maybe there is no code change necessary because functionally if we assumed the JWT was an opaque token (ie: we were not trying to decode it), then it should be functionally the same with the kratos token

Reply:
imho to make this functional we need to remove this https://github.com/GaloyMoney/galoy-cli/blob/main/src/main.rs#L84-L86 . Here we are trying to check for a valid JWT. I tried to this way and it worked for me.

Reply:
yup this should not longer work with the new way we authenticate query. this should be removed.

Reply:
@nicolasburtey if we remove the part mentioned by @thevaibhav-dixit then how to authenticate, like based on the length of JWT?

Reply:
And the error in checks coming due to old version of remove_dir_all that is 0.5.3 as it is supported with version above 0.8.0. but there is no crate "remove_dir_all" in cargo.toml that need to be updated. how to solve this issue can you please guide me?